autogen.sh

```diff
@@ -8,7 +8,8 @@ args="\
 --sysconfdir=/etc \
 --localstatedir=/var \
 --prefix=/usr \
---enable-silent-rules"
+--enable-silent-rules \
+--with-fallback-capaths=/tmp/swupd_test_certificates"
```

Comment:
Do we need to set the fallback-capaths as default in a dev env? Afaik all tests are skiped by default, right? I'd add a comment that you need this flag set in build time if you need to run those tests

Reply:
Yes, '--with-fallback-capaths' is not required when the tests are skipped. I will remove it from autogen.sh and make a comment.